lib/smart_answer_flows/minimum-wage-calculator-employers/questions/past_payment_date.govspeak.erb

```diff
@@ -3,6 +3,7 @@
 <% end %>
 
 <% options(
+  "2017-04-01": "Apr 2017 - Sep 2017",
```

Comment:
@ikennaokpala This option needs to be for "Oct 2016 - Mar 2017".
"Apr 2017 - Sep 2017" will the current period that's checked in the previous question.